packages/common/src/index.ts

```diff
@@ -508,11 +509,20 @@ export type DbtColumnLightdashMetric = {
     sql?: string;
     hidden?: boolean;
     round?: number;
+    format?: string;
 };
 
+export type ResultRow = {
+    [col: string]: {
+        [value: string]: {
+            raw: any;
+            formatted: string;
```

Comment:
If there is no format we return the raw value so the formatted should be `any` right ? 🤔